src/tests/multihost/sssd/testlib/common/qe_class.py

```diff
@@ -191,6 +191,17 @@ def dnf_uninstall(self, package):
                                raiseonerr=False)
         return cmd.returncode
 
+    def detect_files_provider(self):
+        """
+        Detect the SSSD's files provider feature 
+        :param: None
+        :return bool: Return code of the ls command
```

Comment:
'Return code of the ls command' -> 'Returns "True" if "files provider" is supported"

Reply:
updated.